docker/bash.sh

```diff
@@ -138,6 +138,8 @@ CONTAINER_NAME=
 # "${REPO_DIR}".  The consistent directory for Jenkins is currently
 # necessary to allow cmake build commands to run in CI after the build
 # steps.
+# TODO(oss-team): Figure out a better way to keep the same path
+# between build and testing stages: https://github.com/apache/tvm/issues/11952
```

Comment:
I might be missing some context but what's wrong with this way of keeping it all the same?

Reply:
The assumption on using ./docker/bash.sh is that it will map the same path to the docker, but as I was trying to use it in another CI with ci_hexagon, we encounter an issue because of this assumption. @areusch suggested to add a comment here so we can potentially fix this later.

Reply:
could we include a link to a GH issue with more specifics? I think TODO(oss-team) is kind of a dead-end here.

Reply:
there is a github issue link in the comment in code.

Reply:
sure, but what I'm saying is that oss-team doesn't have meaning in TVM. we can just remove it or place the link inside the TODO(), e.g. `TODO(https://github.com/apache/tvm/issues/11952)`

Reply:
done.